### PR TITLE
Update README.md to follow current situation

### PR DESCRIPTION
**Pull request by shirakia**
Updated old README.md

Reviewing with this URL is recommended
https://github.com/camenergydatalab/EnergyDataSimulationChallenge/pull/67/files?w=1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # EnergyDataSimulationChallenge
 
 Welcome to EnergyDataSimulationChallenge!
-This project is to challenge the analysis of big energy production data.
+This project is to challenge the analysis/development of big energy production data.
```
Comment (hankehly) [inline README.md:4]: "This challenge project allows applicants to demonstrate their ability to analyze and develop software the makes use of big energy production data."

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 We prepared several challenges.
-You can try either of them or maybe all :)
+You can try one of them or maybe all :)
```
Comment (hankehly) [inline README.md:6]: "Please complete one (or more) of the several challenges we have prepared."

File: README.md [hunk 3]
```diff
@@ -24,10 +24,13 @@ Your pull-request is always welcome.
 
 ### Reward
 
-* Winner will be kindly offered the position for Paid Internship. (3-month programme, competitive salary.)
-* Accommodation (private room) and your desk at Co-Working space in Cambridge, UK will be provided for FREE if you wish to work with us here.
-* Or, you can still work from your own place remotely.
-* After Successful 3-month programme, we could offer other positions (permanent position, longer contract, additional intern etc.)
+* For Full-Time Employment
+    * Chance to final interview with CTO and engineers.
```
Comment (hankehly) [inline README.md:28]: A chance to earn a final interview with CTO and engineers.

File: README.md [hunk 4]
```diff
@@ -24,10 +24,13 @@ Your pull-request is always welcome.
...
+* For Paid Internship
+    * Winner will be kindly offered the position for Paid Internship. (3-month programme, competitive salary.)
```
Comment (hankehly) [inline README.md:30]: Winner will be offered a paid internship position (3 month program with competitive salary)

**Comment by shirakia** [comment]
Hank "sensei" ...!

File: README.md [hunk 5]
```diff
@@ -24,10 +24,13 @@ Your pull-request is always welcome.
...
+    * Accommodation (private room) and your desk at Co-Working space in London will be provided for FREE if you wish to work with us here.
+    * Or, you can still work from your own place remotely.
```
Comment (hankehly) [inline README.md:32]: Or, you may work remotely from a location of your choice.

**Review by hankehly — APPROVED** [review]
@shirakia The meaning is already clear; but my comments include more common expressions.

File: README.md (hunk 4 above)
Comment (shirakia) [reply README.md:30]: `programme` is kind of British stuff

**Comment by shirakia** [comment]
@hankehly Yeah, this repository has been created and updated by Japanese, Italian, French and German engineers but British/American haha. Can you create a new PR for updating README?

**Pull request merged**